apps/frontend/src/network/Routes.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import {
+  Switch,
+  Route,
+  useRouteMatch,
+  useHistory,
+  useLocation,
+} from 'react-router-dom';
+import { NetworkPage } from '@asap-hub/react-components';
+
+import ProfileList from './ProfileList';
+import TeamList from './TeamList';
+
+const Network: React.FC<{}> = () => {
+  const { path } = useRouteMatch();
+  const { search } = useLocation();
+  const history = useHistory();
+  const currentUrlParams = new URLSearchParams(search);
+
+  const toggleOnChange = (pathname: string) => () => {
+    history.push({ pathname, search: currentUrlParams.toString() });
+  };
+  const searchOnChange = (newQuery: string) => {
+    currentUrlParams.set('query', newQuery);
+    history.replace({ search: currentUrlParams.toString() });
+  };
+  return (
+    <Switch>
+      <Route
+        exact
+        path={`${path}/users`}
+        render={() => (
```

Comment:
`react-router-dom` now (idk since when exactly, actually) supports `children`! It's a much more readable substitute for `render` :tada: